### DISCUSSION TITLE
Only T bug

### DISCUSSION THREAD

**Discussion by Dazzyb85** [AntonyCorbett/OnlyT#517] [General]
Are you aware of any bugs on Only T? 

We are experiencing it bugging and not showing us the the timer control. Even on task master we cannot shut it off. 
It shows on timer screen and meeting countdown is on display. But acts as if programme has closed and unable to reopen unless restart pc

**Comment by AntonyCorbett** [comment]
@Dazzyb85 has the main window simply been minimised? Are there any OnlyT icons on the Windows taskbar? If so, just click the icon to restore the main window.

By "Task Master", do you mean "Task Manager" or the Taskbar right-click menu? You should always be able to use "End Task" to close OnlyT using the Task Manager, but if you are referring to the Taskbar menu, then you won't be able to close all OnlyT windows while a timer is running. Please provide more details.

Another issue may be that the position or size of the main window has been stored incorrectly so that when OnlyT launches it displays its main window "off screen". To fix this, close OnlyT, delete the "%AppData%\OnlyT\1" folder and restart OnlyT.

Hope this helps. Please get back to me if you need more assistance.

Regards
Antony

**Comment by Simo1973** [comment]
A me capita la stessa cosa, non si vede nel Task Manager, si apre in miniatura e quando tenti di ingrandirlo sparisce, ma funziona perché con l'app remota sullo smartphone lo comandi e sul browser vedi il tempo.

Ho provato a cancellare la cartella suggerita ma non è cambiato nulla.

Uso WIN11 su un I3 di decima generazione con 16GB di RAM

**Comment by Simo1973** [comment]
Io ho risolto: appena avviato era in miniatura, ho ingrandito la finestra tirando il lato in masso a destra e non mi crea più il problema.
